src/uint_common.c

```diff
@@ -58,3 +58,16 @@ void reverseString(char *const str, uint32_t length) {
         str[j] = c;
     }
 }
+
+int bytes_to_string(char *out, size_t outl, const void *value, size_t len) {
+    if (strlcpy(out, "0x", outl) != 2) {
+        goto err;
+    }
+    if (format_hex(value, len, out + 2, outl - 2) == -1) {
+        goto err;
+    }
+    return 0;
+err:
+    *out = '\0';
```

Comment:
```
if (outl >= 1) {
    out[0] = '\0';
}
```
Probably useless but might prevent a nasty error if we call with outl==0

Reply:
Better to check `outl` at the begining of the function, to ensure the possibility, at least to write `0x`.

Reply:
Fine by me, one or the other